src/field_theory/adjoin.lean

```diff
@@ -171,11 +178,25 @@ end
 lemma algebra_adjoin_le_adjoin : algebra.adjoin F S ≤ (adjoin F S).to_subalgebra :=
 algebra.adjoin_le (subset_adjoin _ _)
 
-lemma adjoin_le_algebra_adjoin (inv_mem : ∀ x ∈ algebra.adjoin F S, x⁻¹ ∈ algebra.adjoin F S) :
-  (adjoin F S).to_subalgebra ≤ algebra.adjoin F S :=
-show adjoin F S ≤
-  { neg_mem' := λ x, (algebra.adjoin F S).neg_mem, inv_mem' := inv_mem, .. algebra.adjoin F S},
-from adjoin_le_iff.mpr (algebra.subset_adjoin)
+lemma adjoin_eq_algebra_adjoin (inv_mem : ∀ x ∈ algebra.adjoin F S, x⁻¹ ∈ algebra.adjoin F S) :
```

Comment:
It might be too much for this PR, but is this lemma true with the weaker assumption `(inv_mem : ∀ x ∈ S, x⁻¹ ∈ algebra.adjoin F S)`?

Reply:
Nevermind, it's not true. Adjoining x and x^(-1) to K results in a ring that is not a field.